api/WorkerNavigator.json

```
@@ -109,10 +109,10 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": "46"
+              "version_added": null
```

```suggestion
              "version_added": false
```
https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#version_added says that `null` means *“Support unknown”* while `false` means *“No support”*. So this should be `false`, right?